IndexedDB/idbobjectstore_createIndex15-autoincrement.htm

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>IDBObjectStore.createIndex() - AutoIncrement in Compound Index</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="support.js"></script>
+<script>
+  indexeddb_test(
+    function(t, db, txn) {
```

Comment:
You can use ES6 syntax like `const` and `(t, db, txn) => { ... }` if that makes the test more readable. Optional, though.